pkg/metrics/metrics.jsx

```diff
@@ -1405,7 +1412,10 @@ const PCPConfigDialog = ({
 
                     let real_redis;
                     let redis_name;
-                    if (s_redis_server.exists) {
+                    if (s_valkey.exists && s_valkey.unit?.UnitFileState !== 'masked') {
+                        real_redis = s_valkey;
+                        redis_name = "valkey.service";
```

Comment:
Should we maybe make the service name configurable via the manifest as well? Probably not, since we want Cockpit to pick up and be happy with "redis.service" on Fedora when the "redis" package is installed, right?

Reply:
Right -- I think the .service detection is better left as dynamic runtime test. E.g. if someone has already installed valkey on RHEL or Debian, we want to prefer this as well. The manifest is only for detecting the on-demand installation package name, as that's too expensive and brittle to do dynamically.

Reply:
Yep, makes sense.